### PR TITLE
Upgrade HerdDB to 0.16, Pulsar Client to 2.5.2, add demo config for diskless-cluster mode

### PR DESCRIPTION
**Pull request by eolivelli**
### Motivation

Since HerdDB 0.16 you can store all of the data on BookKeeper, this is called "diskless-cluster" mode.
So you will be able to run PulsarManager + HerdDB on a stateless pod, all of the data will be stored on the same BK+ZK cluster as your Pulsar installation. 

### Modifications

- Upgrade HerdDB to 0.16.1.
- Upgrade Apache Pulsar Admin Client to 2.5.2 (that officially supports BK 4.10)
- Use pulsar-client-admin-original instead of pulsar-client-admin, because pulsar-client-admin contains a shaded version of CirceChecksum library from BK that is not compatibile with BK 4.10, needed by HerdDB in order to run in diskless-cluster  mode.
- Configure gradle to put application.properties inside pulsar-manager.tar, this way it is super easy to change the configuration of pulsar-manager backend without recompiling the package (no need to use Gradle for System administrator)
- Add an example JDBC URI to boot HerdDB in diskless-cluster mode

### Verifying this change

- [ ] Make sure that the change passes the `./gradlew build` checks.

### CODE REVIEW

**Comment by eolivelli** [comment]
@tuteng @sijie  PTAL

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -39,12 +39,15 @@ spring.datasource.driver-class-name=herddb.jdbc.Driver
 # HerdDB - local in memory-only
 #spring.datasource.url=jdbc:herddb:local
 # HerdDB - start embedded server, data persisted on local disk (directory 'dbdata'), listening on localhost:7000
-spring.datasource.url=jdbc:herddb:server:localhost:7000?server.start=true&server.base.dir=dbdata
+#spring.datasource.url=jdbc:herddb:server:localhost:7000?server.start=true&server.base.dir=dbdata
+# HerdDB - start embedded server 'diskless-cluster' mode, WAL and Data persisted on Bookies, Metadata on ZooKeeper in '/herd', listening on localhost:7000
+#spring.datasource.url=jdbc:herddb:zookeeper:localhost:2181?server.start=true&server.base.dir=dbdata&server.mode=diskless-cluster&server.node.id=localhost
```
Comment (aluccaroni) [inline src/main/resources/application.properties:44]: Is it normal that in the new config we don't have any "spring.datasource.url" config uncommented?

Comment (eolivelli) [reply src/main/resources/application.properties:44]: good catch !
I have restored the commented line

thanks @aluccaroni

**Comment by tuteng** [comment]
LGTM, Thanks, @eolivelli

**Comment by tuteng** [comment]
@aluccaroni PTAL

**Pull request merged**